language-plutus-core/src/Language/PlutusCore/CBOR.hs

```diff
@@ -272,3 +273,90 @@ instance Serialise UnknownDynamicBuiltinNameError
 instance (Closed uni, uni `Everywhere` Serialise, Serialise ann) => Serialise (InternalTypeError uni ann)
 instance (Closed uni, uni `Everywhere` Serialise, Serialise ann) => Serialise (TypeError uni ann)
 instance (Closed uni, uni `Everywhere` Serialise, Serialise ann) => Serialise (Error uni ann)
+
+
+{- Serialising the unit annotation takes up space: () is converted to
```

Comment:
Make this a proper Note?

Nit: for long multiline doc comments I think we're mostly adding a line break and then not indenting the main body of the comment: makes editing easier.